app/models/crime_application.rb

```diff
@@ -3,6 +3,7 @@ class CrimeApplication < ApplicationRecord
 
   has_one :applicant, dependent: :destroy
   has_one :partner, dependent: :destroy
+  has_one :ioj, through: :case, dependent: :destroy
```

Comment:
I think we can get rid of this, so if we ever feel inclined to use this as a shorthand instead of going through `case`, it fails and we can consider our options.